src/main/java/ladder/backEnd/controller/LadderController.java

```diff
@@ -0,0 +1,26 @@
+package ladder.backEnd.controller;
```

Comment:
패키지명은 관례에 따라 소문자로 작성해보면 좋을 것 같습니다.
```suggestion
package ladder.backend.controller;
```